aten/src/ATen/native/native_functions.yaml

```diff
@@ -14211,6 +14211,11 @@
 - func: linalg_solve(Tensor A, Tensor B, *, bool left=True) -> Tensor
   python_module: linalg
 
+- func: _spsolve(Tensor A, Tensor B, *, bool left=True) -> Tensor
+  python_module: sparse
+  dispatch:
+    SparseCsrCUDA: _sparse_csr_linear_solve
```

Comment:
I am writing it in this way because `linalg_solve` is already (implicitly) using the `CompositeImplicitAutograd` dispatch key. `CompositeImplicitAutograd` has higher priority than `SparseCsrCUDA`, so adding `SparseCsrCUDA ` dispatch rule under `linalg_solve` would have no effect in my previous attempts.